solutions/swb-reference/package.json

```diff
@@ -19,6 +19,8 @@
     "cdk": "cdk",
     "cdk-deploy": "rush build && rushx compile && cdk deploy --all --require-approval never --outputs-file ./src/config/${STAGE}.json",
     "cdk-deploy:noBuild": "rushx compile && cdk deploy --all",
+    "cdk-synth": "rush build && rushx compile && cdk synth",
```

Comment:
running cdk-synth and cdk-deploy requires two full builds, is that necessary?

Reply:
if we use cdk-synth following cdk-deploy in the pipeline, we should convert that to the cdk-deploy:noBuild to reduce time.

Reply:
@htztomic yeah I understand your concern, but we don't use `cdk-synth` followed by `cdk-build`. Even if you re-run the build it won't rebuild it. It will skip the builds if there is nothing new to build.

Reply:
good point